packages/remix/src/utils/integrations/opentelemetry.ts

```diff
@@ -0,0 +1,53 @@
+import { RemixInstrumentation } from 'opentelemetry-instrumentation-remix';
+
+import { defineIntegration } from '@sentry/core';
+import { SEMANTIC_ATTRIBUTE_SENTRY_OP, addOpenTelemetryInstrumentation, spanToJSON } from '@sentry/node';
+import type { Client, IntegrationFn, Span } from '@sentry/types';
+import type { RemixOptions } from '../remixOptions';
+
+const _remixIntegration = ((options?: RemixOptions) => {
+  return {
+    name: 'Remix',
+    setupOnce() {
+      addOpenTelemetryInstrumentation(
+        new RemixInstrumentation({
```

Comment:
m: Let's re-write this based on the changes done in https://github.com/getsentry/sentry-javascript/pull/12213, which means extracting this into a `instrumentRemix` method which is called in `setupOnce`. You'll have to put the options into module scope so they can be updated later...!

Reply:
Done 👍